<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Debugger::Internal::BreakpointDialog</class>
 <widget class="QDialog" name="BreakpointDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Breakpoint Properties</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelType">
       <property name="text">
        <string>Breakpoint type:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBoxType"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelFileName">
       <property name="text">
        <string>File name:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="Utils::PathChooser" name="pathChooserFileName"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelLineNumber">
       <property name="text">
        <string>Line number:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEditLineNumber"/>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="checkBoxEnabled">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelEnabled">
       <property name="text">
        <string>Enabled:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QCheckBox" name="checkBoxUseFullPath">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelUseFullPath">
       <property name="text">
        <string>Use full path:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="labelAddress">
       <property name="text">
        <string>Address:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lineEditAddress"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="labelFunction">
       <property name="text">
        <string>Function:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="lineEditFunction"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="labelCondition">
       <property name="text">
        <string>Condition:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="lineEditCondition"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="labelIgnoreCount">
       <property name="text">
        <string>Ignore count:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="lineEditIgnoreCount"/>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="labelThreadSpec">
       <property name="text">
        <string>Thread specification:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLineEdit" name="lineEditThreadSpec"/>
     </item>
     <item row="10" column="1">
      <widget class="QCheckBox" name="checkBoxTracepoint">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="labelTracepoint">
       <property name="text">
        <string>Tracepoint only:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BreakpointDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BreakpointDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
